code/tests/functional/backend_api/tests/with_data/test_conversation_custom.py

```diff
@@ -100,7 +100,11 @@ def test_post_makes_correct_call_to_get_search_index(
                 "Accept": "application/json;odata.metadata=minimal",
                 "Api-Key": app_config.get("AZURE_SEARCH_KEY"),
             },
-            query_string="api-version=2023-11-01",
+            query_string=(
+                "api-version=2023-11-01"
+                if not app_config.get("AZURE_SEARCH_USE_INTEGRATED_VECTORIZATION")
+                else "api-version=2023-10-01-Preview"
```

Comment:
Nit: can we change this to use "if app_config.get("AZURE_SEARCH_USE_INTEGRATED_VECTORIZATION")" rather than "if not", will make it easier to read.